tests/codeception/common/_support/FixtureHelper.php

```diff
@@ -58,6 +58,30 @@ public function globalFixtures()
     }
 
     /**
+     * Workaround used to load custom fixtures for the test
+     * @link https://github.com/Codeception/Codeception/issues/2258
+     * @link https://github.com/yiisoft/yii2/issues/5575
+     * @param array $fixtureConfig
+     */
+    public function loadFixturesForTest(array $fixtureConfig)
+    {
+        $fixtures = ArrayHelper::merge(
```

Comment:
> can you fix it?

I can. But for my mind we should make `globalFixtures`, `fixtures`, `createFixtures`  public instead of `getFixture`. This way we can be more flexible with fixture management.